WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3081,6 +3136,16 @@ public void onMediaUploaded(OnMediaUploaded event) {
 
     // FluxC events
 
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onMediaListFetched(OnMediaListFetched event) {
+        // no op - we don't need to check anything just now, but declaring the method so it's
+        // clear we make a request to FetchMedia in this class.
```

Comment:
@mzorz do we still want this comment here like how we have logic below? LMK 🙏

Reply:
good catch  - removed in 13e5bfd2ca